rudder-web/src/main/scala/com/normation/rudder/web/components/RuleCompliance.scala

```diff
@@ -190,6 +210,14 @@ class RuleCompliance (
                 x: { show: true }
               , y: { show: true }
             }
+          , onrendered: function () {
+              if(firstLoad){
+                  document.getElementsByClassName('c3-bar-12')[0].classList.add('c3-bar-highlighted');
+                  $$('#selectedPeriod b').text(x[x.length-1]);
+                  console.log('ok');
+                  firstLoad = false;
+              }
+          }
         } );
         $$('#changesChart').html(chart.element);
```

Comment:
Maybe you should do:

```
document.getElementsByClassName('c3-bar-12')[0].classList.add('c3-bar-highlighted');
$$('#selectedPeriod b').text(x[x.length-1]);
```

and remove the onrendered function and the firstload var